File: aplicacaomvc/Deloitte_NF.Web/Models/Produto.cs

namespace Deloitte_NF.Models
{
    /// <summary>
    /// Classe Produto
    /// </summary>
    public class Produto
    {
        /// <summary>
        /// Obtém e informa o Sequencial do Produto
        /// </summary>
        /// <value>Sequencial do Produto</value>
        public int IdProduto { get; set; }

        /// <summary>
        /// Obtém e informa os Dados do Fornecedor
        /// </summary>
        /// <value>Dados do Fornecedor</value>
        public Fornecedor Fornecedor { get; set; }

        /// <summary>
        /// Obtém e informa o Nome do Produto
        /// </summary>
        /// <value>Nome do Produto</value>
        public string NomeProduto { get; set; }

        /// <summary>
        /// Obtém e informa o Valor do Produto
        /// </summary>
        /// <value>Valor do Produto</value>
        public double ValorProduto { get; set; }
    }
}

File: aplicacaowebforms/Negocio/Entidades/Produto.vb
Public Class Produto
    Inherits Generico

    Private _fornecedor As Fornecedor
    Private _valorProduto As Double

    Public Property Fornecedor As Fornecedor
        Get
            Return _fornecedor
        End Get
        Set(value As Fornecedor)
            _fornecedor = value
        End Set
    End Property

    Public Property ValorProduto As Double
        Get
            Return _valorProduto
        End Get
        Set(value As Double)
            _valorProduto = value
        End Set
    End Property
End Class